src/features/cartographie/pages/lieux-mediation-numerique-list/lieux-mediation-numerique-list.page.ts

```diff
@@ -82,7 +86,12 @@ export class LieuxMediationNumeriqueListPage {
       this.boundingBox$()
     ),
     this.route.paramMap
-  ]).pipe(map(toLieuxFilteredByDepartement), map(toLieuxWithLieuToFocus), tap(this.setInitialState), map(toLieux));
+  ]).pipe(
+    map(toLieuxFilteredByDepartement),
+    map(toLieuxWithLieuToFocus),
+    tap(this.setInitialState),
+    map((lieux) => toLieux(this._localisation, lieux))
```

Comment:
Changer la signature de la fonction pour pouvoir écrire ça de cette manière (avec curryfication) :
```suggestion
    map(toLieux(this._localisation))
```